lib/util.js

```diff
@@ -24,11 +24,37 @@ function normalizeVersions (versions) {
 }
 
 function bufferizeVersion (version) {
+  if (typeof version !== 'string' && typeof version !== 'number')
+    throw new Error('invalid version type.')
+
+  if (typeof version === 'string') {
+    return hexStringToBuffer(version)
+  } else if (typeof version === 'number') {
+    // expects BE uint
+    return beUIntToBuffer(version)
+  }
+}
```

Comment:
```js
if (typeof version === 'string') return hexStringToBuffer(version)
if (typeof version === 'number') return beUIntToBuffer(version)
throw new Error('invalid version type.')
```

Reply:
Good call on this :+1: will update with that